DevGraphs.md

```diff
@@ -0,0 +1,48 @@
+# Preview: Dev Graphs
+
+We have wanted to make Studio's tools helpful to folks during local development for a long time. Up until now, Studio has largely been tailored to work with graphs already running in production.
+
+Dev Graphs are **a new type of graph** that users can create in Studio. They have special properties that have been optimized for development-time concerns.
+
+### Dev Graph Properties
+
+- Dev graphs are set up and kept up-to-date with schema introspection.
+- Dev graphs are unique to you –– only you can see and use the dev graphs you create.
+- Dev graphs watch for changes to your server so they can update themselves near-instantaneously when your schema changes.
+- Dev graphs are transient and can be considered "disposable" in your workflow (e.g. you could make a dev graph per branch).
+- Dev graphs are sandboxes that are safe to experiment with, and completely separate from your other graphs in Studio.
+- Every tool in dev graphs is completely free to use, and always will be.
+
+Dev graphs contain a subset of Studio's full feature set right now:
+
+- Explorer
+- Schema Reference
+- Schema Changelog
+
+We will be introducing more of Studio's tools for regular graphs to dev graphs over time as we add support for local development to those features.
```

Comment:
I think we could skip saying this for now - let's see if people ask for that, but no reason to make promises here.